core/client/src/main/java/alluxio/hadoop/AbstractFileSystem.java

```diff
@@ -456,6 +456,8 @@ public void initialize(URI uri, org.apache.hadoop.conf.Configuration conf) throw
     List<URIStatus> statuses;
     try {
       statuses = sFileSystem.listStatus(uri);
+    } catch (FileDoesNotExistException fnte) {
```

Comment:
Could you change this to `FileDoesNotExistException e` to be consistent with our style?